regparser/tree/xml_parser/paragraph_processor.py

```diff
@@ -117,6 +120,20 @@ def process(self, xml, root):
             markers = [node.label[0] for node in nodes]
             constraints = self.additional_constraints()
             depths = derive_depths(markers, constraints)
+
+            if not depths:
+                logging.warning("Could not derive paragraph depths."
+                                "Retrying with relaxed constraints.")
+                deemphasized_markers = [deemphasize(m) for m in markers]
+                additional_constraints = [optional_rules.star_new_level,
+                                          optional_rules.limit_paragraph_types(
+                                           mtypes.lower, mtypes.upper,
+                                           mtypes.ints, mtypes.roman,
+                                           mtypes.em_ints, mtypes.em_roman,
+                                           mtypes.stars, mtypes.markerless)]
+                depths = derive_depths(deemphasized_markers,
+                                       additional_constraints)
```

Comment:
No that won't work, when you remove the emphasis it will fail unless you also relax the constraints (due to the depth_type_inverses constraint).